cargo.toml

```diff
@@ -0,0 +1,5 @@
+[workspace]
```

Comment:
Also, one other thing, there is already a `Cargo.toml` file, and the duplicate with different casing is screwing up on my machine. The standard is a capital C

Reply:
Ah this is also messing things on my machine. Well it was...